Makefile

```diff
@@ -6,7 +6,17 @@
 #
 
 MK_DIR :=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
-.PHONY: test test-release-tools
+SED := sed
+YQ := $(shell go env GOPATH)/bin/yq
```

Comment:
I guess this won't work with [multi-path GOPATHs](https://golang.org/cmd/go/#hdr-GOPATH_environment_variable) - but then I don't think we've ever seen one in use, so I don't see it as a blocker - we have GOPATH usage like that all over the repos...